swing/src/net/sf/openrocket/gui/dialogs/ComponentAnalysisDialog.java

```diff
@@ -186,69 +182,72 @@ public void stateChanged(ChangeEvent e) {
 		panel.add(tabbedPane, "spanx, growx, growy");
 
 
-		// Create the CP data table
-		cpTableModel = new ColumnTableModel(
+		// Create the Longitudinal Stability (CM vs CP) data table
+		longitudeStabilityTableModel = new ColumnTableModel(
 
 				//// Component
 				new Column(trans.get("componentanalysisdlg.TabStability.Col.Component")) {
 					@Override
 					public Object getValueAt(int row) {
-						Object c = cgData.get(row)[0];
-						if (c instanceof Rocket) {
-							return trans.get("componentanalysisdlg.TOTAL");
-						}
+						Object c = stabData.get(row).name;
+//						if (c instanceof Rocket) {
```

Comment:
commented out code can be removed

Reply:
previous versions of this dialog labeled this row as "Total" instead of the rocket name.  

I left this in, in case people objected and wished for the previous behavior.
Seeing no objections: removed.